app/controllers/users_controller.rb

```diff
@@ -14,4 +14,20 @@ def edit
         .by_start_at
     )
   end
+
+  def sort
+    @guiders = User
+               .guiders
+               .includes(:groups)
+  end
+
+  def sort_update
+    guider_order = params[:order_users][:guider_order]
+    ActiveRecord::Base.transaction do
```

Comment:
Can you put some breathing space around this? Just a line of whitespace before and after should make this somewhat easier to scan.